docs/data-sources/l3out_redistribute_policy.md

```diff
@@ -0,0 +1,56 @@
+---
+# Documentation generated by "gen/generator.go"; DO NOT EDIT.
+subcategory: "L3Out"
+layout: "aci"
+page_title: "ACI: aci_l3out_redistribute_policy"
+sidebar_current: "docs-aci-data-source-aci_l3out_redistribute_policy"
+description: |-
+  Data source for L3out Redistribute Policy
+---
+
+# aci_l3out_redistribute_policy #
+
+Data source for L3out Redistribute Policy
+
+## API Information ##
+
+* Class: [l3extRsRedistributePol](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/l3extRsRedistributePol/overview)
+
+* Supported in ACI versions: 4.2(1i) and later.
+
+* Distinguished Name Formats
+  - `uni/tn-{name}/out-{name}/rsredistributePol-[{tnRtctrlProfileName}]-{src}`
+
+## GUI Information ##
+
+* Location: Tenants -> Networking -> L3Outs -> Policy -> Main -> Route Profile for Redistribution
+
+## Example Usage ##
+
+```hcl
+
+data "aci_l3out_redistribute_policy" "example_l3_outside" {
+  parent_dn                  = aci_l3_outside.example.id
+  source                     = "direct"
+  route_control_profile_name = aci_route_control_profile.example.name
+}
+
+```
+
+## Schema
+
+### Required
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_l3_outside](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/l3_outside) ([l3extOut](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/l3extOut/overview))
+* `source` (src) - (string) The source of the L3out Redistribute Policy object.
```

Comment:
Add options to required attributes with options in data sources?

Reply:
added